src/components/staking/PageValidator.vue

```diff
@@ -34,29 +34,40 @@
       </div>
       <tr class="li-validator">
         <td class="data-table__row__info">
-          <Avatar
-            v-if="!validator.picture || validator.picture === 'null'"
-            class="li-validator-image"
-            alt="generic geometric symbol - generated avatar from address"
-            :address="validator.operatorAddress"
-          />
-          <img
-            v-else-if="validator.picture"
-            :src="validator.picture"
-            :alt="`validator logo for ` + validator.name"
-            class="li-validator-image"
-          />
-          <div class="validator-info">
-            <h3 class="li-validator-name">
-              {{ validator.name }}
-            </h3>
-            <div v-if="delegation.amount">
-              <h4>{{ delegation.amount | fullDecimals }}</h4>
-              <h5 v-if="rewards">
-                +{{ rewards.amount | fullDecimals | noBlanks }}
-              </h5>
+          <div class="li-validator-name-row">
+            <Avatar
+              v-if="!validator.picture || validator.picture === 'null'"
+              class="li-validator-image"
+              alt="generic geometric symbol - generated avatar from address"
+              :address="validator.operatorAddress"
+            />
+            <img
+              v-else-if="validator.picture"
+              :src="validator.picture"
+              :alt="`validator logo for ` + validator.name"
+              class="li-validator-image"
+            />
+            <div class="validator-info">
+              <h3 class="li-validator-name">
+                {{ validator.name }}
+              </h3>
+              <div v-if="delegation.amount">
+                <h4>{{ delegation.amount | fullDecimals }}</h4>
+                <h5 v-if="rewards">
+                  +{{ rewards.amount | fullDecimals | noBlanks }}
+                </h5>
+              </div>
             </div>
           </div>
+          <span
+            v-if="!validator.picture || validator.picture === 'null'"
+            class="no-img-info"
+          >
+            Would you like us to display your logo?&nbsp;<a
+              href="mailto:contact@lunie.io"
```

Comment:
intercom instead?

Reply:
I think you are right. Intercom is quicker and overall easier to use

Reply:
intercom works on desktop 😛 - there is likely a different problem